packages/grid/_modules_/grid/components/GridPagination.tsx

```diff
@@ -60,26 +60,12 @@ export const GridPagination = React.forwardRef<
   );
 
   const handlePageChange = React.useCallback(
```

Comment:
```suggestion
  const handlePageChange = React.useCallback<TablePaginationProps['onPageChange']>(
```

Reply:
A lot better indeed

Reply:
It won't work with `onRowsPerPageChange`, I tried. Maybe the typings in the core are wrong.